lib/config/config_test.go

```diff
@@ -215,6 +215,7 @@ func checkStaticConfig(c *check.C, conf *FileConfig) {
 	c.Assert(conf.Proxy.Enabled(), check.Equals, true)     // Missing "proxy_service" section must lead to 'true'
 	c.Assert(conf.Proxy.Disabled(), check.Equals, false)   // Missing "proxy_service" does NOT mean it's been disabled
 	c.Assert(conf.AdvertiseIP.String(), check.Equals, "10.10.10.1")
+	c.Assert(conf.PidFile, check.Equals, "/var/run/teleport.pid")
```

Comment:
`Pid` -> `PID` according to https://github.com/golang/go/wiki/CodeReviewComments